site/src/plugins/genai-samples-docs-plugin.ts

```diff
@@ -0,0 +1,134 @@
+import { Plugin } from '@docusaurus/types';
+import fs from 'fs';
```

Comment:
I think you could also import from `fs/promises` to get rid of `fs.promises. ...`